content/en/docs/concepts/scheduling/configure-scheduler.md

```diff
@@ -0,0 +1,217 @@
+---
+title: How to Configure Your Kubernetes Scheduler
+reviewers:
+- wgliang
+content_template: templates/concept
+weight: 30
+toc_hide: false
+---
+
+{{% capture overview %}}
+
+This is third article of how to use and extend the Kubernetes scheduler series, the whole series of artices are,
```

Comment:
Suggestion (fixed a small typo too): This is the third article on how to use and extend the Kubernetes scheduler series. The series of articles are listed below:

Reply:
Thank you very much, Shavidissa! I will update them.